.vitest/vitest.shared.ts

```diff
@@ -4,11 +4,12 @@ import { configDefaults, defineConfig } from "vitest/config";
 export const sharedConfig = defineConfig({
   test: {
     alias: {
-      "~test": join(__dirname, "."),
+      "~test": join(__dirname, "./src"),
```

Comment:
TODO: will need to update tsconfig to make this work in test files too, but will do it in the next PR which will use the prool setup to update 1 test